great_expectations/expectations/metrics/query_metrics/query_table.py

```diff
@@ -34,27 +34,35 @@ def _sqlalchemy(
     ) -> List[dict]:
         query: Optional[str] = metric_value_kwargs.get(
             "query"
-        ) or cls.default_kwarg_values.get("query")
+        ) or metric_value_kwargs.get("unexpected_rows_query")
+
+        batch_ref = (
+            "batch"
+            if metric_value_kwargs.get("unexpected_rows_query")
+            else "active_batch"
+        )
```

Comment:
There is a different pattern than this in v1, but it's not worth the effort to backport that pattern. This is likely one of the last v0 releases.